pkgs/tools/audio/yabridge/default.nix

```diff
@@ -51,19 +51,19 @@ let
       repo = "vst3sdk";
       rev = version;
       fetchSubmodules = true;
-      sha256 = "1fqpylkbljifwdw2z75agc0yxnhmv4b09fxs3rvlw1qmm5mwx0p2";
+      sha256 = "sha256-4oLOa6kVB053Hrq7BBbZFdruAXuqnC944y5Kuib1F7s=";
```

Comment:
```suggestion
      hash = "sha256-4oLOa6kVB053Hrq7BBbZFdruAXuqnC944y5Kuib1F7s=";
```

Reply:
When `fetchSubmodules` is set under `fetchFromGitHub`, it uses `fetchgit` instead of `fetchzip`, but that currently doesn't support the `hash` attribute yet: https://github.com/NixOS/nixpkgs/pull/79987